backend/experiment/rules/matching_pairs.py

```diff
@@ -79,8 +80,8 @@ def next_round(self, session):
                 title='Score',
                 final_text='Can you score higher than your friends and family? Share and let them try!',
                 button={
-                    'text': 'Play again', 'link': '{}/{}'.format(
-                        settings.CORS_ORIGIN_WHITELIST[0], session.experiment.slug)
+                    'text': 'Play again', 'link': '/{}'.format(
+                        session.experiment.slug)
```

Comment:
```suggestion
                    'text': 'Play again',
                    'link': f'/{session.experiment.slug}'
```

Perhaps this would make it a bit more readable